_posts/builtin/2022-02-23-container-orchestration-provider.md

```diff
@@ -0,0 +1,159 @@
+  
+- **Internal Ports** - This field accepts a comma separated list of ports which will be internally exposed on the spun up docker container.
+  
+- **External Ports** - This field accepts a comma separated list of ports which will be externally exposed on the host machine.
+  
+- **Privileged Mode** - This flag if enabled will give the container root capabilities to all devices on the host system. Please be aware that setting this flag can be dangerous, and must only be used in exceptional situations.
+  
+- **Environment Variables (optional)** - This field accepts a comma separated list of environment variables, which will be set inside the container when spun up.
+  
+- **Volume Mount (optional)** - This field accepts a comma separated list of system-to-container file mounts. This allows for the container to access files on the host machine.
+  
+- **Peripheral Device (optional)** - This field accepts a comma separated list of device paths. This parameter allows for devices to be passed though from the host to the container.
+
+- **Logger Type** - This field provides a dropdown selection of supported container logging drivers.
+  
+- **Logger Parameters (optional)** - This field accepts a comma separated list of logging parameters. More information can be found in the [Docker logger documentation](https://docs.docker.com/config/containers/logging/configure/#supported-logging-drivers).
+  
+
+After specifying container parameters, ensure to set **Enabled** to **true** and press **Apply**. Docker will then pull the respective image locally, or through docker hub, spin up and start the container. If the gateway is reset (power cycled), and the container and Docker-service is set to **enabled**, Kura will automatically start the container again upon startup. 
+![Supported-Container-Configuration]({{ site.baseurl }}/assets/images/builtin/container_orchestrator/container_configuration.png)
+
+## Stopping the container
+
+To stop the container without deleting the component, set the **Enabled** field to **false**, and then press **Apply**. This will delete the running container, but leave this component available for running the container again in the future. If you want to completely remove the container and component, press the **Delete** button to the top right of the screen, and press **Yes** on the confirmation dialog.
+![Stop-Container]({{ site.baseurl }}/assets/images/builtin/container_orchestrator/stop_container.png)
+
+## Container Management Dashboard
+
+The Container Orchestration service also provides the user with a intuitive container dashboard. This dashboard shows all containers running on a gateway, including containers created with Kura and those created manually though the command line interface. To utilize this dashboard the **org.eclipse.container.orchestration.provider** (Docker-API) must be enabled, and the dashboard can be opened by navigating to **Device > Docker Containers**.
```

Comment:
those created manually **though** the command line -> those created manually th**r**ough the command line